docs/man/git-lfs-ext.1.ronn

```diff
@@ -0,0 +1,25 @@
+git-lfs-ext(1) - View extension details
+==============================================================
+
+## SYNOPSIS
+
+`git lfs ext list` [<name>...]
+
+## DESCRIPTION
+
+Git LFS extensions enable the manipulation of files streams
+during smudge and clean.
+
+## EXAMPLES
+
+* List details for all extensions
+
+    `git lfs ext list`
+
+* List details for the specified extensions
+
+    `git lfs ext list 'foo' 'bar'`
+
```

Comment:
I thought we had agreed that list would be optional, so that "git lfs ext" shows all extensions.  My only other thought is that if we want to support additional verbs later, like add, we should make sure that it's not legal to say "git lfs ext foo" now.